src/scportrait/tools/parse/_parse_phenix.py

```diff
@@ -462,19 +461,24 @@ def replace_missing_images(self):
     def define_copy_functions(self):
         """Define function for copying depending on if symlinks should be used or not"""
         if self.export_symlinks:
+
             def copyfunction(input, output):
                 try:
-                    if platform.system() == 'Windows':
+                    if platform.system() == "Windows":
+                        print(
```

Comment:
here I would recommend using the warning class:

```
import warnings
warnings.warn("Warning...........Message")
```

This ensures that the user is correctly informed about the behaviour and the print statement isn't lost anywhere

Reply:
Replaced print with warning statement.

Reply:
I can't see the commit yet. Have you pushed it?

Reply:
Should be there now